### PR TITLE
Update harness-git-ops-application-set-tutorial.md

### PR DESCRIPTION
**Pull request by michaelcretzman**
I was asked to add an explanation of wave deployments to the PR Pipeline doc by Cat Palmer to support a blog article. Cat wants to ensure that someone searching for "wave deployments" in our docs finds it discussed as part of GitOps. It belongs in the PR Pipeline topic as that the blog article discusses wave deployments in the GitOps context.

blog draft (not sure who wrote it): https://docs.google.com/document/d/1gUoabLk8bN3odoAN-dh_-6Wejcp6PNaeutTPvPPOgOY/edit

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ x] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* request by Cat Palmer

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by samarthwadhwa — DISMISSED** [review]

File: docs/continuous-delivery/cd-gitops/harness-git-ops-application-set-tutorial.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ In this tutorial, we'll demonstrate two major use cases:
 
 :::note
 
-**New to ApplicationSets and PR Pipelines?** See the [ApplicationSet and PR Pipeline Summary](harness-git-ops-application-set-tutorial.md#application-set-and-pr-pipeline-summary) below.
+**New to ApplicationSets and PR Pipelines?** See the [ApplicationSet and PR Pipeline Summary](#applicationset-and-pr-pipeline-summary) below.
```
Comment (aimurphy) [inline docs/continuous-delivery/cd-gitops/harness-git-ops-application-set-tutorial.md:26]:
```suggestion
**New to ApplicationSets and PR Pipelines?** Go to [ApplicationSet and PR Pipeline Summary](#applicationset-and-pr-pipeline-summary).
```

Style guide says to use "Go to" before links.

File: docs/continuous-delivery/cd-gitops/harness-git-ops-application-set-tutorial.md [hunk 2]
```diff
@@ -572,6 +572,18 @@ When you deploy a Harness PR Pipeline, you simply indicate what target environme
 
 ![](./static/harness-git-ops-application-set-tutorial-62.png)
 
+#### Wave deployments
+
+You often here the term wave deployments used when PR Pipelines are discussed.
```
Comment (aimurphy) [inline docs/continuous-delivery/cd-gitops/harness-git-ops-application-set-tutorial.md:577]:
```suggestion
You often hear the term wave deployments used when PR Pipelines are discussed.
```

File: docs/continuous-delivery/cd-gitops/harness-git-ops-application-set-tutorial.md [hunk 3]
```diff
@@ -572,6 +572,18 @@ When you deploy a Harness PR Pipeline, you simply indicate what target environme
...
+
+A wave deployment is a deployment strategy in Continuous Delivery that involves releasing changes to a portion of users at a time, rather than all users at once. Typically, this is done using separate cloud regions for each target environment.
+
+Wave deployments help reduce the risk of deployment failures and allow for quick recovery. The changes are rolled out in waves, typically starting with a group of users in one region and gradually expanding to the entire user base across all regions. This approach allows for a more controlled and monitored rollout of changes, improving the overall quality and stability of the deployment process.
+
+With Harness GitOps, you can implement wave deployments by creating multiple environments for your application, one for each cloud region, and gradually promoting changes from one environment to the next. This way, you can test changes in a safe and controlled manner before releasing them to the entire user base.
```
Comment (aimurphy) [inline docs/continuous-delivery/cd-gitops/harness-git-ops-application-set-tutorial.md:583]:
```suggestion
With Harness GitOps, you can implement wave deployments by creating multiple environments for your application: one environment for each cloud region. Then, gradually promote changes from one environment to the next. This way, you can test changes in a safe and controlled manner before releasing them to the entire user base.
```

Suggested revision to break up long sentence. Also wondering about the use of "safe"; does it unintentionally imply that other deployment techniques are not safe?

**Review by aimurphy — APPROVED** [review]
approve w/ assumption you will consider my suggestions

**Pull request merged**